models/article.models.js

```diff
@@ -28,3 +28,8 @@ exports.updateArticleById = async (id, update) => {
   }
   return article.rows[0];
 };
+
+exports.fetchArticles = async () => {
+  const article = await db.query("SELECT * FROM articles ORDER BY created_at desc;");
+  return article.rows;
```

Comment:
I'd be tempted to rename this variable from article to `result` - as the rows property _**is**_ the articles in this case